tutorials/Tutorial17_Audio.ipynb

```diff
@@ -0,0 +1,360 @@
+    "\n",
+    "# - Converts a file into text and performs basic cleaning (TextConverter node)\n",
+    "text_converter = TextConverter(\n",
+    "    remove_numeric_tables=True\n",
+    ")\n",
+    "indexing_pipeline.add_node(text_converter, name=\"text_converter\", inputs=[\"classifier.output_1\"])\n",
+    "\n",
+    "# - Pre-processes the text by performing splits and adding metadata to the text (Preprocessor node)\n",
+    "preprocessor = PreProcessor(\n",
+    "    clean_whitespace=True, \n",
+    "    clean_empty_lines=True, \n",
+    "    split_length=100, \n",
+    "    split_overlap=50,\n",
+    "    split_respect_sentence_boundary=True\n",
+    ")\n",
+    "indexing_pipeline.add_node(preprocessor, name=\"preprocessor\", inputs=[\"text_converter\"])\n",
+    "\n",
+    "# - Writes the resulting documents into the document store (ElasticsearchDocumentStore node from the previous cell)\n",
+    "indexing_pipeline.add_node(document_store, name=\"document_store\", inputs=[\"preprocessor\"])\n",
+    "\n",
+    "# Then we run it with the documents and their metadata as input\n",
+    "indexing_pipeline.run(file_paths=file_paths, meta=files_metadata)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Querying\n",
+    "   \n",
+    "Now we will create a pipeline very similar to the basic `ExtractiveQAPipeline` of Tutorial 1,\n",
+    "with the addition of a node that converts our answers into audio files!\n",
+    "\n",
+    "Note that if you're doing document retrieval, Haystack provides and a `DocumentToSpeech` node\n",
+    "that can be used in a similar way (or in an indexing pipeline)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from haystack import Pipeline\n",
+    "from haystack.nodes import BM25Retriever, FARMReader, AnswerToSpeech\n",
+    "\n",
+    "retriever = BM25Retriever(document_store=document_store)\n",
+    "reader = FARMReader(model_name_or_path=\"deepset/roberta-base-squad2\", use_gpu=True)\n",
+    "answer2speech = AnswerToSpeech(model_name_or_path=\"espnet/kan-bayashi_ljspeech_vits\", generated_audio_dir=Path(__file__).parent / \"audio_answers\")\n",
+    "\n",
+    "audio_pipeline = Pipeline()\n",
+    "audio_pipeline.add_node(retriever, name=\"Retriever\", inputs=[\"Query\"])\n",
+    "audio_pipeline.add_node(reader, name=\"Reader\", inputs=[\"Retriever\"])\n",
+    "audio_pipeline.add_node(answer2speech, name=\"AnswerToSpeech\", inputs=[\"Reader\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ask a question!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "is_executing": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# You can configure how many candidates the reader and retriever shall return\n",
```

Comment:
...the Reader and Retriever should return.

Reply:
This also was copied straight from Tut1. I will fix that too.